config/pipeline.yaml

```diff
@@ -101,6 +101,22 @@ storage:
 
 runtimes:
 
+  lab-kontron:
+    lab_type: legacy.lava_rest
+    url: 'https://lavalab.kontron.com/'
+    filters:
+      - passlist:
+          plan:
+            - baseline
```

Comment:
Those aren't actually used in the current system, and not needed: we have `rules:` for filtering on kernel trees/branches/versions and other small things, but scheduler entries directly associate jobs with runtimes, so there's no filtering needed here.

Reply:
right, but if I remove it here, where should I add the `rules:`? on the `baseline:` definition?

```yaml
  baseline: &baseline-job
    template: baseline.jinja2
    kind: test
    rules:
      tree:
      - 'android'
```

How can I define which tests should be run on the tree I'm adding?

Reply:
Most of those can be "inherited" from the kbuild job: in the `scheduler` entries, you'll associate test jobs with a parent node/job, so test jobs will only run on trees the parent node can use.

In this case, you'll add those rules to the `kbuild-gcc-10-armel-android*` jobs you're defining; that way, those kbuild nodes will only run on the android tree, and baseline jobs depending on those will only run on this tree as well.

Reply:
I added this change for now, please let me know if it is not correct